### ISSUE TITLE
Calling getInstalledApplications returns empty list

### ISSUE THREAD

**Issue by aswinmohanme** [g123k/flutter_plugin_device_apps#49]
I have this code in my widget
```
class _SelectCustomAppScreenState extends State<SelectCustomAppScreen> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      backgroundColor: Theme.of(context).colorScheme.background,
      body: SafeArea(
        child: Container(
          padding: EdgeInsets.all(kScreenPadding),
          child: Column(
            crossAxisAlignment: CrossAxisAlignment.start,
            children: <Widget>[
              SizedBox(height: s_4),
              HeadingText("Select Custom App"),
              SizedBox(height: s_2),
              Expanded(
                child: FutureBuilder(
                  future: DeviceApps.getInstalledApplications(
                      includeSystemApps: true, onlyAppsWithLaunchIntent: true),
                  builder: (BuildContext context, var snapshot) {
                    if (snapshot.hasData) {
                      print(snapshot.data);
                      return Center(child: CircularProgressIndicator());
                    } else {
                      return Center(child: CircularProgressIndicator());
                    }
                  },
                ),
              ),
            ],
          ),
        ),
      ),
    );
  }
}

```

This should return the list of installed applications in the `snapshot.data` but here is the output on my console 

```
Reloaded 4 of 698 libraries in 537ms.
I/flutter (31817): [DeviceApps] type '_InternalLinkedHashMap<dynamic, dynamic>' is not a subtype of type 'Map<String, Object>'
I/chatty  (31817): uid=10290(me.aswinmohan.nophone) Thread-2 identical 122 lines
I/flutter (31817): [DeviceApps] type '_InternalLinkedHashMap<dynamic, dynamic>' is not a subtype of type 'Map<String, Object>'
I/flutter (31817): []
```

Here is my `pubspec.yaml` for reference
```
environment:
  sdk: ">=2.7.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter

  # The following adds the Cupertino Icons font to your application.
  # Use with the CupertinoIcons class for iOS style icons.
  cupertino_icons: ^0.1.3

  date_format: ^1.0.8
  intent: ^1.4.0
  google_fonts: ^1.1.0
  tailwind_colors: ^0.1.0
  battery: ^1.0.1
  device_apps: ^1.1.1+1
  hive: ^1.4.2
  hive_flutter: ^0.3.1

dev_dependencies:
  flutter_test:
    sdk: flutter

  hive_generator: ^0.7.1
  build_runner: ^1.10.1

```

**Comment by aswinmohanme** [comment]
I have rebuilt the Application thinking that the issue was Native code not getting recompiled, but the issue is not that.

**Comment by g123k** [comment]
I am working on a fix right now, should be published anytime soon

**Comment by g123k** [comment]
Fixed in 1.1.2
